.circleci/config.yml

```diff
@@ -629,7 +652,7 @@ jobs:
             - verify-codestyle
             - verify-license
             #disabled to bring common 23.3.1 on top of 23.3.1-rc.1
-#            - verify-common-sdk-version
+            #            - verify-common-sdk-version
```

Comment:
@mapbox/navigation-android should this still be commented? I see that common version is different now.